### DISCUSSION TITLE
Hungarian training data

### DISCUSSION THREAD

**Discussion by hekkr000** [openai/whisper#710] [Ideas]
Today I've tested the v2 model vs the v1 and found that in a lot of cases the v2 one tends to just spit out this text: " Feliratok az Amara.org közösségétől". I think there might be a lot of invalid data scraped from this site: amara.org. I mostly tested the models with music, here are a few examples:

https://youtu.be/1BI54w6T_Uo
https://youtu.be/M5CwqYQNRcY

I think it would be beneficial to look this string up and clean the training data.
Otherwise, I've seen a large improvement in Hungarian, whenever the whisper was willing to create real transcripts.

**Comment by SilverGreen93** [comment]
Hi,
I randomly get the same message in Hungarian in kdenlive using Hungarian Large model

![image](https://github.com/openai/whisper/assets/24845602/99cc01a5-c69d-4493-8ba6-e8d67ed2ca7b)

Don't know what the issue is, but seems quite random to get stuck like this.

**Comment by snefi** [comment]
i have same... error...
